pkg/util/helper/work.go

```diff
@@ -83,6 +85,40 @@ func GetWorksByLabelsSet(c client.Client, ls labels.Set) (*workv1alpha1.WorkList
 	return workList, c.List(context.TODO(), workList, listOpt)
 }
 
+// GetWorksBySameResourceBindingAndLabelsSet get WorkList by matching same Namespace and same Name.
+func GetWorksBySameResourceBindingAndLabelsSet(c client.Client, bindingNamespace, bindingName string) (*workv1alpha1.WorkList, error) {
```

Comment:
This func name doesn't seem clearly, I think we can directly use `GetWorksByBindingNamespaceName`.